github-issue-opener/README.md

```diff
@@ -13,7 +13,7 @@ it like this:
 # TODO: pre-reqs like ko/google providers.
 
 module "issue-opener" {
-  source = "github.com/chainguard-dev/enforce-events//github-issue-opener/iac"
```

Comment:
This extra slash is necessary: c.f https://www.terraform.io/language/modules/sources#modules-in-package-sub-directories

Reply:
haha, of course it is :)